static/js/controllers/contacts.js

```diff
@@ -77,7 +77,9 @@ var _ = require('underscore'),
 
         if (options.skip) {
           $scope.appending = true;
-          options.skip = liveList.count();
+
+          // Round down to the nearest standard page size
+          options.skip = Math.floor(liveList.count() / options.limit) * options.limit;
```

Comment:
This would cause an error if one of the first page entries got deleted while you were looking at it (the code would skip 50 but should skip 49). It might be better to check explicitly for the the home place and add/remove one.

A more robust approach would be to use the [`startkey` approach](http://docs.couchdb.org/en/2.1.1/ddocs/views/pagination.html#paging-alternate-method), but that's much more work.